### ISSUE TITLE
BOINC -- Failed call to GrantUserRight - SeDenyRemoteInteractiveLogonRight

### ISSUE THREAD

**Issue by romw** [BOINC/boinc-ng#766]
**Reported by Ageless on 25 Jan 38937151 08:53 UTC**
Ever since one of the 6.3 versions installing BOINC on a Windows 2000 system generates two pop-ups with a warning during the installation of BOINC. 

Here's what I lifted out of Event Viewer on the error, while installing 6.4.4:
```
Event Type:	Error
Event Source:	MsiInstaller
Event Category:	None
Event ID:	1013
Date:		07/12/08
Time:		17:15:32
User:		TVOUT\Administrator
Computer:	TVOUT
Description:
Product: BOINC -- Failed call to GrantUserRight - SeDenyRemoteInteractiveLogonRight 
Data:
0000: 7b 38 38 39 46 42 35 31   {889FB51
0008: 39 2d 30 31 36 31 2d 34   9-0161-4
0010: 36 41 32 2d 42 46 31 43   6A2-BF1C
0018: 2d 43 42 31 44 45 38 30   -CB1DE80
0020: 38 30 45 45 44 7d         80EED}  
```

It does not stop BOINC from further installing or running, but the error is annoying, none-the-least and can scare off users.

**Comment by romw** [comment]
**Commented by Nicolas on 4 May 38937159 04:53 UTC**
Anyone interested in making a [binary search](wikipedia:binary_search) for the specific 6.3 version that introduced this problem?

**Comment by romw** [comment]
**Commented by Ageless on 30 Dec 38937479 06:13 UTC**
I first reported it happening on 6.3.14, through the Alpha email list. See [this entry](http://lists.ssl.berkeley.edu/mailman/private/boinc_alpha/2008-October/009408.html). 

Lynn W. Taylor reports it separately in [this thread](http://lists.ssl.berkeley.edu/mailman/private/boinc_alpha/2008-October/009586.html).

All on Windows 2000 SP4 (and in my case, all the latest updates).

**Comment by romw** [comment]
**Commented by romw on 4 Jul 38951183 10:13 UTC**
(In [16674])     - WINSETUP: Make sure we don't try and do anything with the
        SeDenyRemoteInteractiveLogonRight user right on Windows 
        2000 or older. Fixes #789
        
    win_build/installerv2/redist/Windows/src/boinccas/
        boinccas.rc
        CAGrantBOINCAdminsRights.cpp
        CAGrantBOINCMasterRights.cpp
        CAGrantBOINCProjectRights.cpp
        CAGrantBOINCProjectsRights.cpp
        CAGrantBOINCUsersRights.cpp
